sql/hive/src/main/scala/org/apache/spark/sql/hive/client/package.scala

```diff
@@ -117,7 +118,8 @@ package object client {
         "org.apache.derby:derby:10.14.1.0"),
       exclusions = Seq("org.apache.calcite:calcite-druid",
         "org.apache.curator:*",
-        "org.pentaho:pentaho-aggdesigner-algorithm"))
+        "org.pentaho:pentaho-aggdesigner-algorithm",
+        "org.apache.hive:hive-vector-code-gen"))
```

Comment:
do other Hive versions like `v2_2` have the same issue?

Reply:
@zhouyifan279 Could you answer @sunchao 's question?

Reply:
Thank you for confirming, @zhouyifan279 .